capstone/src/main/webapp/youtubeScript.js

```diff
@@ -36,42 +37,41 @@ const YOUTUBE_CATEGORIES = {
 
 /** Calls youtube servlet and passes output to perspctive */
 async function callYoutube() {
-  document.getElementById('search-type').innerHTML = ""
+  document.getElementById('search-type').innerHTML = "";
   const channelId = document.getElementById('channelIdForAnalysis').value.replace(/ /g, '');
   if (!channelId) {
     return;
   }
   /** Checks if input is a category, if so directs input to be handled by get trending*/
   if (YOUTUBE_CATEGORIES[channelId] != undefined) {
-    document.getElementById('search-type').innerHTML ="Category Search";
+    document.getElementById('search-type').innerHTML = "Category Search";
     getTrending(YOUTUBE_CATEGORIES[channelId]);
     return;
   }
   /** Checks if input follows channel ID format, if not attempts to convert it to channel ID*/
   var response;
+  var responseJson;
   if (channelId[0] == "U" && channelId[1] == "C" && channelId.length == 24 && isLetter(channelId[channelId.length-1])) {
     response = await fetch('/youtube_servlet?channelId=' + channelId,)
-    response = await response.json();
-    if (response.hasOwnProperty('error')) {
-      alert("Invalid Channel ID");
+    responseJson = await response.json();
+    if (responseJson.hasOwnProperty('error')) {
       inputCommentsToPerspective([]);
       return;
     }
-    document.getElementById('search-type').innerHTML ="Channel ID Search";
+    document.getElementById('search-type').innerHTML = "Channel ID Search";
   } else {
-    const usernameConverterResponse = await fetch('/username_servlet?channelId=' + channelId,)
+    const usernameConverterResponse = await fetch('/youtube_username_servlet?channelId=' + channelId,)
```

Comment:
nit: Let's delete the extra comma at the end of this line and add `;`

Reply:
addressed in "general formatting"